src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  margin-right: 50px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  height: 60px;
+  line-height: 60px;
+  cursor: pointer;
+}
+
+.nav__link {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  margin-left: 20px;
```

Comment:
better add margin-left for `nav__item`, not for link, because you stretching width of parrent element to achieve gap between `nav__item`'s, and it's not correct